release-notes/releases.json

```diff
@@ -93,7 +93,7 @@
         "rps-win-x86": "aspnetcore-runtime-2.1.0-win-x86.zip",
         "rps-mac-x64": "aspnetcore-runtime-2.1.0-osx-x64.tar.gz",
         "rps-linux-x64": "aspnetcore-runtime-2.1.0-linux-x64.tar.gz",
-        "asp-runtime-linux-arm-x64":"aspnetcore-runtime-2.1.0-linux-arm.tar.gz",
+        "asp-runtime-linux-arm":"aspnetcore-runtime-2.1.0-linux-arm.tar.gz",
```

Comment:
This is the 64bit download, right? I see in the storage account that asp is producing

* aspnetcore-runtime-internal-2.1.1-linux-arm.tar.gz
* aspnetcore-runtime-internal-2.1.1-linux-musl-x64.tar.gz
* aspnetcore-runtime-internal-2.1.1-linux-x64.tar.gz

I presume the *linux-arm-tar.gz is 32bit. If this is true, wouldn't it be better to append the release asset with the correct architecture (x64) which will make the eventual addition of the 32 bit asset natural? Otherwise, when that time comes, the thing people have been accessing as "asp-runtime-linux-arm" will change from 64 to 32 bit.

Reply:
`aspnetcore-runtime-2.1.0-linux-arm.tar.gz` is a 32bit release. It's technically arm32v7. IIUC running arm32 from arm64 is optional in the Linux kernel, and in practice some arm64 processors do not support arm32.